CONTRIBUTING.md

```diff
@@ -368,6 +369,27 @@ at the beginning of the line:
 I love it! Very clean code and great tests. lgtm.
 ```
 
+## Continuous Integration
+
+The Kata Containers project has a gating process to prevent introducing
+regressions. When your PR is submitted, a Continuous Integration (CI) system
+will run different checks on different platforms upon your changes. Currently
+Kata uses [Jenkins](http://jenkins.katacontainers.io) and
+[travis](https://travis-ci.org/kata-containers/) for testing your changes.
```

Comment:
Nit: For consistency with Jenkins, you could capitalise this or use the full name "Travis CI". Same comment for the other mention of "travis" below.